utils/rpms/daos.spec

```diff
@@ -4,10 +4,11 @@
 
 %global mercury_version 2.0.1~rc1-1%{?dist}
 %global libfabric_version 1.12.0
+%global __python %{__python3}
```

Comment:
Did something break that requires this?  I'd think the system macros should take care of this.

Reply:
seems its a result of a bad merge

Reply:
actually no, it seems to be already in master, no idea why it was added

Reply:
Yeah.  I was looking at a subset of commits that included the merge commit which showed this as an added line.